tailwind.config.js

```diff
@@ -9,24 +10,26 @@ module.exports = {
 			fontFamily: {
 				lexend: ["Lexend"],
 			},
-			colors: {
-				acm: {
-					blue: "#82aaff",
-					lightpurple: "#c792ea",
-					yellow: "#ffd700",
-					green: "#c3e88d",
-					red: "#ff5370",
-					gray: "#8f93a2",
-					pink: "#f07178",
-					lightblue: "#89ddff",
-					darkgray: "#525975",
-					orange: "#f78c6c",
-					marine: "#80cbc4",
-					purple: "#da70d6",
-					white: "#ffffff",
-					black: "#000000",
-				},
+			backgroundImage: {
+				arrow: "url('/chevron.png')",
 			},
+			acm: {
```

Comment:
this should be under `colors`

Reply:
Oops, must have merged it wrong, corrected in commit